src/reader-activation/index.js

```diff
@@ -422,6 +402,60 @@ function attachNewsletterFormListener() {
 	} );
 }
 
+
+/**
+ * Checkout functions.
+ */
+
+/**
+ * Get the current checkout.
+ *
+ * @return {Object} Checkout data.
+ */
+export function getCheckout() {
+	return store.get( 'checkout' ) || {};
+}
+
+/**
+ * Set the current checkout data.
+ *
+ * @param {Object} data Optiona. Checkout data.
```

Comment:
```suggestion
 * @param {Object} data Optional. Checkout data.
```

Reply:
There's a bit of inconsistency in the doc blocks for the checkout functions regarding spacing and phrasing. Here it starts with "Optional.", other end with "Optional".

Reply:
Oops. Good catch. Fixed the typo and moved the instances of `Optional` to the end in each doc block in https://github.com/Automattic/newspack-plugin/pull/3343/commits/f68ba174858d8213dacda9f734355bd60f9fd638